opentech/static_src/src/app/src/redux/reducers/submissions.js

```diff
@@ -0,0 +1,50 @@
+import {
+    FAIL_LOADING_SUBMISSIONS_BY_ROUND,
+    SET_CURRENT_SUBMISSION_ROUND,
+    START_LOADING_SUBMISSIONS_BY_ROUND,
+    UPDATE_SUBMISSIONS_BY_ROUND,
+} from '@actions/submissions';
+
+const initialState = {
+    currentRound: null,
+    itemsByRound: {},
+    itemsByRoundLoadingError: false,
+    itemsByRoundLoading: false,
+};
+
+export default function submissions(state = initialState, action) {
+    switch(action.type) {
+        case SET_CURRENT_SUBMISSION_ROUND:
+            return {
+                ...state,
+                currentRound: action.id,
+            };
+        case UPDATE_SUBMISSIONS_BY_ROUND:
+            const newData = {};
+            newData[action.roundId] = action.data.results;
+
+            return {
+                ...state,
+                itemsByRound: {
```

Comment:
I would like to see the store de-normalised to be `submissionsByID: {1: {app1}, 2:{app2}}` and `submissionsByRoundID: {1: [1, 2]}` 

We'll then be looking at `submissionsByStatus` at a later date

I've got an example from another project if that helps?